ext/vscode/.vscode/settings.json

```diff
@@ -9,5 +9,7 @@
     // Turn off tsc task auto detection since we have the necessary tasks as npm scripts
     "typescript.tsc.autoDetect": "off",
     // Help the cSpell extension find our configuration files (it doesn't import .yaml by default)
-    "cSpell.import": ["./.vscode/cspell.yaml"]
+    "cSpell.import": ["./.vscode/cspell.yaml"],
+    // Block committing directly to main branch
+    "git.branchProtection": ["main"],
```

Comment:
:-D

Reply:
A nice little quality-of-life thing I decided to add here when I accidentally committed to main :smile: